storage_controller/src/service.rs

```diff
@@ -732,7 +743,30 @@ impl Service {
             std::process::exit(1);
         }
 
-        self.inner.write().unwrap().become_leader();
+        let safekeepers = self.inner.read().unwrap().safekeepers.clone();
+        let sk_schedule_requests = match safekeeper_reconciler::load_schedule_requests(
+            self,
+            &safekeepers,
+        )
+        .await
+        {
+            Ok(v) => v,
+            Err(e) => {
+                tracing::error!(
```

Comment:
Can we not make it fail hard until we get some testing?

Reply:
done